test/Core.Test/Tools/Services/SendServiceTests.cs

```diff
@@ -116,6 +119,93 @@ public async Task SaveSendAsync_DisableHideEmail_DoesntApply_success(SendType se
         await sutProvider.GetDependency<ISendRepository>().Received(1).CreateAsync(send);
     }
 
+    // Disable Send policy check - vNext
+    private void SaveSendAsync_Setup_vNext(SutProvider<SendService> sutProvider, Send send,
+        SendPolicyRequirement sendPolicyRequirement)
+    {
+        sutProvider.GetDependency<IPolicyRequirementQuery>().GetAsync<SendPolicyRequirement>(send.UserId!.Value)
+            .Returns(sendPolicyRequirement);
+        sutProvider.GetDependency<IFeatureService>().IsEnabled(FeatureFlagKeys.PolicyRequirements).Returns(true);
+
+        // Should not be called in these tests
+        sutProvider.GetDependency<IPolicyService>().AnyPoliciesApplicableToUserAsync(
+            Arg.Any<Guid>(), Arg.Any<PolicyType>()).ThrowsAsync<Exception>();
```

Comment:
👍🏻 I love when a test causes a method that shouldn't be called to throw. It's such an elegant way to test the invariant.